### ISSUE TITLE
Multi-CPU version for Windows installation?

### ISSUE THREAD

**Issue by Alphisto** [bootchk/resynthesizer#36]
Hi,

first of all, a big thank you for keeping this most useful plugin alive and working! I read in the readme of the most downloaded Windows v1.0 version this sentence:

> (This is v1.0 of the resynthesizer.  A later version, v2.0, does the same things, only faster, if you have a multicore processor.)

And I was wondering how to install this version? Is it possible to do it the same wasy way of copying files do plug-ins folder as the v1.0? I haven't found the same files I think. Is it necessary to compile it?

Thanks for you time,
Adam Preisler

**Comment by bootchk** [comment]
Yes you would need to compile it.  I don't know why "the most downloaded WIndows version" is not v2.0.  Maybe you should persuade that distributor to build it.  I myself do not build it for Windows, and other people build it for other distributions (Debian?)
The multi-threaded version is not much faster (an 8 core machine does NOT speed it up by a factor of 8, maybe a factor of 2?)  Due to the nature of the algorithm (many passes, each a search.)  Unless you use it a lot, not worth the trouble.

**Comment by SanderBouwhuis** [comment]
Can someone help me with a newer version of Resynthesizer for Windows?
Has anyone compiled it?
Can I compile it myself using Visual Studio, or does it require external dependencies?
